src/renderer/components/dialogs/About.tsx

```diff
@@ -67,6 +71,18 @@ export function DCInfo(_props: any) {
 export default function About({ onClose }: DialogProps) {
   const tx = useTranslationFunction()
 
+  const [coreVersion, setCoreVersion] = useState('')
+  const [sqliteVersion, setSqliteVersion] = useState('')
+
+  useEffect(() => {
+    BackendRemote.rpc.getSystemInfo().then(info => {
+      console.warn({ info })
```

Comment:
I'd use `info` as a log level here. If you want to hide it from the developer by default, use `debug`, that's still not a warning but optional

Reply:
ah good catch it should be removed, I just used it for debugging